contracts/LemonadePoapV1Unique.sol

```diff
@@ -32,24 +32,45 @@ contract LemonadePoapV1Unique is LemonadePoapV1 {
             accessRegistry
         )
     {
-        collection = ICollectionHelpers(collectionHelpers)
-            .createNonfungibleCollection(name, name, symbol);
+        ICollectionHelpers collectionHelpers_ = ICollectionHelpers(
+            collectionHelpers
+        );
 
-        ICollection(collection).setTokenPropertyPermission(
+        collection = collectionHelpers_.createNFTCollection{value: msg.value}(
+            name,
+            name,
+            symbol
+        );
+        collectionHelpers_.makeCollectionERC721MetadataCompatible(
+            collection,
+            tokenURI
+        );
+
+        ICollection collection_ = ICollection(collection);
+
+        collection_.addCollectionAdmin(address(this));
```

Comment:
I guess it worth to add 
`collection_.addToCollectionAllowList(address(this));`

To mint tokens to the collection, address should be in the allow list